CHANGELOG.md

```diff
@@ -22,6 +22,21 @@ This project adheres to the Node [default version scheme](https://docs.npmjs.com
 - Internal: Update FaceTec SDK on Auth step from 9.4.11 to 9.4.12
 - Internal: Show connection error screen on active video upload errors
 
+## [9.1.0] - 2022-09-16
+
+- Internal: Migrated onfido/react-webcam fork to typescript and absorbed it into this repo.
+- Public: Added support for cs_CZ, ro_RO and pl_PL locales
+- Internal: Add utility tool for locales
+- Internal: Add Logger module
+- Internal: Remove woopra
+- Internal: Migrate analytics from `/v3` to `/v4`
+- Internal: Place cross device SMS feature behind a feature flag
+- Internal: Add `disableAnalyticsCookies` option
+- Public: Increase capture resolution for useLiveDocumentCapture mode, to prevent image quality issues.
+- Internal: Integrate Passive Signals module
+
+## [9.0.0] - 2022-09-05
+
```

Comment:
Humm this looks off to me @DannyvanderJagt @zoeradkani 🤔

Reply:
@ethan605 the date or the internal update?